src/background.xml

```diff
@@ -5,66 +5,76 @@
       Transport Capacity measurement was known to be well beyond
       our capabilities. Even at the time that Framework for IP
       Performance Metrics 
-      <xref target="RFC3148" /> was written we knew that we didn't
+      <xref target="RFC2330" /> was written we knew that we didn't
       fully understand the problem. Now, by hindsight we understand
-      why BTC is such a hard problem:
+      why assessing BTC is such a hard problem:
       <list style="symbols">
 
         <t>TCP is a control system with circular dependencies -
         everything affects performance, including components that
-        are explicitly not part of the test.</t>
+        are explicitly not part of the test (for example, the host 
+        processing power is not in-scope of path performance tests).</t>
 
-        <t>Congestion control is an equilibrium process, such that
-        transport protocols change the packet transfer statistics
-        (raise the packet loss ratio and/or RTT) to conform to
-        their behavior. By design TCP congestion control keeps
+        <t>TCP congestion control seeks equilibrium through a process 
+        of measuring packet transfer metrics (packet loss ratio and/or
+        mean RTT) and modifying packet transmission characteristics in response. By design TCP congestion control keeps
         raising the data rate until the network gives some
-        indication that it is full by dropping or ECN CE marking
-        packets. If TCP successfully fills the network (e.g. the
-        bottleneck is 100% utilized) the packet loss and ECN CE
-        marks are mostly determined by TCP and how hard TCP drives
-        the network at that rate and not by the network itself.</t>
-
-        <t>TCP's ability to compensate for network flaws is
-        directly proportional to the number of roundtrips per
+        indication that its capacity has been exceeded by dropping 
+        packets (or marking packets CE, Congestion Experienced on ECN-capable connections). If a TCP sender successfully reaches network 
+        capacity of a connection, (e.g. the bottleneck is 100% utilized),
+        then packet losses and ECN CE marks are mostly determined by 
+        the TCP sender and how aggressively it seeks additional capacity 
+        from the network, not the network itself.</t>
+
+        <t>TCP's ability to compensate for network impairments is
+        directly proportional to the number of send-ACK roundtrip exchanges per
         second (i.e. inversely proportional to the RTT). As a
-        consequence a flawed subpath may pass a short RTT local
+        consequence a subpath with limiting bottleneck may pass a short RTT local
```

Comment:
No.  

How about s/flawed/high packet loss rate/  The point is the network is NOT full but has spurious losses that look like congestion.   Such a link is a show stopper at long RTTs and inconsequential a short RTT's.